assets/sass/components/_frontpage-cards.scss

```diff
@@ -66,6 +66,20 @@
 			@include focus-with-outline-only;
 		}
 	}
+	.widecard {
+		display: flex;
+		flex-direction: column;
+		width: 47%;
+		border: solid 1px #ccc;
+		border-radius: 4px;
+		margin: 8px;
+		margin-bottom: 20px;
+		padding: 4px;
+
+		@media (max-width: 669px) {
```

Comment:
The pixel boundaries in media queries should usually use the Boostrap Sass variables. `669px` doesn't really match any of the default boundaries. Why is this value used here?

Examples:
* https://github.com/twbs/bootstrap-sass/blob/6b0ca5025c64f2e5696c8719cfce7148c78d0a50/assets/stylesheets/bootstrap/_variables.scss#L335-L338
* https://github.com/apluslms/a-plus/blob/df64e846d83436bccd92e310662f449a4ff74210/assets/sass/layout/_footer.scss#L21
* https://getbootstrap.com/docs/3.4/css/#sass

Reply:
No reason, just reused the value used earlier in the same file in the lack of better knowledge. Using now grid-float-breakpoint-max as suggested.